src/pages/Defi/components/StakingCard.tsx

```diff
@@ -56,7 +57,7 @@ export const StakingCard = ({
   const handleClick = () => {
     isConnected
       ? history.push({
-          pathname: `/defi/${type}/${provider}/deposit`,
+          pathname: `/defi/${type}/${provider}/withdraw`,
```

Comment:
We've changed it so clicking a Yearn vault will _always_ open to the `withdraw` route.

Should this only be when the vault has expired?

Reply:
I think this makes more sense to open the withdrawal route as on the user's defi assets page they'd probably click on the vault if they wanted to withdraw.

Reply:
as discussed here this is fine for now

https://discord.com/channels/554694662431178782/886041582422356018/939655008314597416